problem-solver/cxx/commonModule/handler/NumberHandler.cpp

```diff
@@ -21,7 +21,7 @@ ScAddr commonModule::NumberHandler::findNumberNode(const double & number)
   ScStreamPtr numberAsStream = ScStreamConverter::StreamFromString(numberAsString);
 
   ScAddrSet candidateList = this->context->SearchLinksByContent(numberAsStream);
-  SC_LOG_DEBUG("Found " << std::to_string(candidateList.size()) << " sc-link with content " << numberAsString);
+  SC_LOG_DEBUG("NumberHandler: Found " << std::to_string(candidateList.size()) << " sc-link with content " << numberAsString);
```

Comment:
Implement the getClassNameForLog() method and use it instead of repeating the class name everywhere. Fix it in all locations